### PR TITLE
tidy-html5: migrate to Conan v2

### PR DESCRIPTION
**Pull request by valgur**
TODO

- [x] v1 test_package fails to find DLL

### CODE REVIEW

**Comment by valgur** [comment]
Closing temporarily to avoid unnecessary load on the CI. Will reopen when I'm actively working on the PR again.

File: recipes/tidy-html5/all/conanfile.py [hunk 1]
```diff
@@ -1,17 +1,22 @@
...
+from conan.tools.files import apply_conandata_patches, copy, export_conandata_patches, get, rm, rmdir
+from conan.tools.microsoft import is_msvc
+
+required_conan_version = ">=1.53.0"
 
 
 class TidyHtml5Conan(ConanFile):
     name = "tidy-html5"
+    description = "The granddaddy of HTML tools, with support for modern standards"
     license = "W3C"
```
Comment (RubenRBS) [inline recipes/tidy-html5/all/conanfile.py:14]: Just checked, and the library does not seem to actually be licensed under these terms:

https://spdx.org/licenses/W3C.html differs substantially from https://github.com/htacg/tidy-html5/blob/next/README/LICENSE.md

**Review by RubenRBS — CHANGES_REQUESTED** [review]

File: recipes/tidy-html5/all/conanfile.py (hunk 1 above)
Comment (valgur) [reply recipes/tidy-html5/all/conanfile.py]: Indeed. Turns out their license actually exists as a separate SPDX liecense ID: [`HTMLTIDY`](https://spdx.org/licenses/HTMLTIDY.html)

**Review by RubenRBS — APPROVED** [review]
TIL about the license! Thanks a lot for the quick response :)

**Review by czoido — APPROVED** [review]

**Pull request merged**